libnmstate/nm/dns.py

```diff
@@ -158,7 +158,11 @@ def get_dns_config_iface_names(acs_and_ipv4_profiles, acs_and_ipv6_profiles):
     for nm_ac, ip_profile in chain(
         acs_and_ipv6_profiles, acs_and_ipv4_profiles
     ):
-        if ip_profile.props.dns or ip_profile.props.dns_search:
+        if (
+            ip_profile.props.dns
+            or ip_profile.props.dns_search
+            or ip_profile.props.dns_options
+        ):
```

Comment:
This seems to me not related to the commit message, maybe worth another commit ?

Reply:
Moved to new commit.